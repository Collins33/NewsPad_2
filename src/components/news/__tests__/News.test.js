import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { News } from "../index";

describe("News", () => {
  it("renders without crashing given the right props", () => {
    const props = {
      news: [{}],
      getNewsArticles: jest.fn(),
      isLoading: false
    };
    const wrapper = shallow(<News {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
