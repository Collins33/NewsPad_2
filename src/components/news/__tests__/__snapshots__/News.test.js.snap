// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News renders without crashing given the right props 1`] = `
<div>
  <div>
    <div>
      <SelectBar
        categories={
          Array [
            Object {
              "value": "business",
            },
            Object {
              "value": "bitcoin",
            },
            Object {
              "value": "politics",
            },
            Object {
              "value": "sports",
            },
            Object {
              "value": "cars",
            },
            Object {
              "value": "technology",
            },
            Object {
              "value": "entertainment",
            },
            Object {
              "value": "education",
            },
            Object {
              "value": "health",
            },
            Object {
              "value": "science",
            },
            Object {
              "value": "general",
            },
          ]
        }
        handleSelectBarClick={[Function]}
      />
      <div>
        <NewsArticle
          checkNewsImage={[Function]}
          email={null}
          getSingleNewsArticle={[Function]}
          news={
            Array [
              Object {},
            ]
          }
        />
        <Paginator
          currentNews={
            Array [
              Object {},
            ]
          }
          handlePaginatorClick={[Function]}
          news={
            Array [
              Object {},
            ]
          }
        />
      </div>
    </div>
  </div>
</div>
`;
